Makefile

```diff
@@ -120,12 +136,18 @@ deploy-dry-run: $(VIRTUAL_ENV) $(PACKAGED_TEMPLATE_FILE)
 
 
 .PHONY: deploy                                                                          ## Deploys Artifacts to S3 Bucket
-deploy: $(VIRTUAL_ENV) $(PACKAGED_TEMPLATE_FILE)
-	@$(MAKE) cfn-deploy stack_name=cz-$(FEATURE_NAME) template_file=$(PACKAGED_TEMPLATE_FILE)
-	# @bucket=`$(MAKE) describe | grep -v -e '^make' | jq -re '.Stacks[0].Outputs | map(select(.OutputKey == "BucketName")) | .[0].OutputValue'` && \
-  # version=`git rev-list --count HEAD` && \
-  # aws s3 sync services s3://$${bucket}/services/v$${version} && \
-  # aws s3 sync services s3://$${bucket}/services/latest
+deploy: $(PACKAGED_TEMPLATE_FILE) package-sam-apps
+	@. ./project.sh && cz_assert_profile && \
+	$(MAKE) cfn-deploy stack_name=cz-$(FEATURE_NAME) template_file=$(PACKAGED_TEMPLATE_FILE)
+	version=`git rev-list --count HEAD` && \
+	for cfn in $(CFN_TEMPLATES) ; do \
+		aws s3 cp $${cfn} s3://$(BUCKET)/v$${version}/$${cfn} && \
+		aws s3 cp $${cfn} s3://$(BUCKET)/latest/$${cfn} ; \
```

Comment:
oh, is this our approach to versioning the customer templates?

Reply:
I'm assuming this is semver because it's a public repo - is it?  If not, consider dropping the `v` on line 144

Reply:
It's not semver. This is how we version our cloudformation catalog and I really like it ... I can add an additional hardcoded "major.minor" part just to "play nice".